src/main/java/core/basesyntax/service/FruitOperationServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitOperationDao;
+import core.basesyntax.model.FruitOperation;
+
+public class FruitOperationServiceImpl implements FruitOperationService {
+    private FruitOperationDao fruitOperationDao;
+
+    public FruitOperationServiceImpl(FruitOperationDao fruitOperationDao) {
+        this.fruitOperationDao = fruitOperationDao;
+    }
+
+    @Override
+    public void createNewFruitOperation(String fruit) {
+        FruitOperation fruitOperation = new FruitOperation();
+        fruitOperation.setFruit(fruit);
+        fruitOperationDao.add(fruitOperation);
```

Comment:
The method `createNewFruitOperation` is quite specific and seems to be doing more than one thing: creating a `FruitOperation` and adding it to the DAO. Consider splitting the creation and the addition of the operation into separate methods to adhere to the Single Responsibility Principle.